### DISCUSSION TITLE
Acid and ORP Daily Limits Reached

### DISCUSSION THREAD

**Discussion by davthomaspilot** [tagyoureit/nodejs-poolController#734] [Q&A] [answered]
How do I clear and set the limits for daily dos of ORP and Acid?

![image](https://user-images.githubusercontent.com/6830886/233861742-0b20c958-d1a1-46eb-9134-25093269d1d6.png)

**Comment by johnny2678** [comment, marked as answer]
Just set a limit in Settings - > Chemistry -> REMChemControl (or whatever you named your ChemController)
ph:
![image](https://user-images.githubusercontent.com/8219743/233863070-840468db-5176-412c-be3f-5bb86ab014b2.png)

orp:
![image](https://user-images.githubusercontent.com/8219743/233863096-2b18672a-714f-433c-b940-651e1cc6c483.png)

Not - ORP daily limit only applies if you are dosing liquid Cl from a tank - just like pH.  if you have an SWG, make sure the `Use Chlorinator` checkbox is checked and the error should clear.

@rstrouse - pls correct me if i'm mistaken.

**Reply by davthomaspilot** [reply]
Those don't show up for me:
![image](https://user-images.githubusercontent.com/6830886/233865991-0c675ebd-0bb6-4dfe-937f-b5ef98cc90fc.png)

![image](https://user-images.githubusercontent.com/6830886/233866006-002142e0-a099-40df-bfa2-ce6ca6f0239d.png)

**Reply by davthomaspilot** [reply]
Oh, because I had manual checked instead of by time or volume.

Sorry and thanks!
